pom.xml

```diff
@@ -66,15 +66,15 @@ Note: The maven build currently needs to be started from the vavr root dir
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <assertj.core.version>3.6.2</assertj.core.version>
         <eclipse.lifecycle.mapping.version>1.0.0</eclipse.lifecycle.mapping.version>
-        <java.version>1.8</java.version>
+        <java.version>9</java.version>
         <jmh.version>1.18</jmh.version>
         <junit.version>4.12</junit.version>
         <gwt.version>2.8.0</gwt.version>
         <maven.build-helper.version>1.12</maven.build-helper.version>
-        <maven.bundle.version>3.2.0</maven.bundle.version>
+        <maven.bundle.version>3.3.0</maven.bundle.version>
```

Comment:
If you separate these in another PR - so that it stays Java 8 & 9 compatible - I'm sure that will be merged :)